features/importing/archivist.feature

```diff
@@ -110,13 +110,63 @@ Feature: Archivist bulk imports
       Then 1 email should be delivered to "otwstephanie@thepotionsmaster.net"
 
   Scenario: Claim a work and create a new account in response to an invite
-  # TODO
+
+    Given I have an archivist "elynross"
+      And the default ratings exist
+      And account creation is enabled
+    When I am logged in as "elynross"
+      And I go to the import page
+      And I import the work "http://cesy.dreamwidth.org/154770.html" by "randomtestname" with email "otwstephanie@thepotionsmaster.net"
```

Comment:
Probably worth taking this opportunity to remove the personal email addresses of former volunteers, while we're at it!